src/Core/Services/OrganizationService.cs

```diff
@@ -1,19 +1,28 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Net;
 using System.Threading.Tasks;
 using Bit.Core.Abstractions;
+using Bit.Core.Exceptions;
 using Bit.Core.Models.Data;
 using Bit.Core.Models.Domain;
+using Bit.Core.Models.Request;
+using Bit.Core.Models.Response;
+using Bit.Core.Utilities;
 
 namespace Bit.Core.Services
 {
     public class OrganizationService : IOrganizationService
     {
         private readonly IStateService _stateService;
+        private readonly IApiService _apiService;
+        readonly LazyResolve<ILogger> _logger = new LazyResolve<ILogger>();
```

Comment:
I don't see the logger being used, if that's the case remove it (if for some reason this would be needed, it should be injected through the ctor)